scipy/sparse/tests/test_construct.py

```diff
@@ -320,6 +322,10 @@ def test_vstack(self):
                      expected)
         assert_equal(construct.vstack([A.tocsr(),B.tocsr()], dtype=np.float32).dtype,
                      np.float32)
+        assert_equal(construct.vstack([A.tocsr(),B.tocsr()],
+                                      dtype=np.float32).indices.dtype, np.int32)
+        assert_equal(construct.vstack([A.tocsr(),B.tocsr()],
```

Comment:
The recommendation is to have one space after a comma. (Sorry for not being clear!)